packages/devtools_app/.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
is this comment supposed to be there?

Reply:
Yes, they contain usable information.

Reply:
okay, I'm surprised this isn't causing issues with json validation for this file